src/main/com/example3/ArrayRNG.java

```diff
@@ -0,0 +1,37 @@
+package main.com.example3;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class ArrayRNG {
+
+    public static void main(String[] args) {
+
+        int array[] = new int[49];
```

Comment:
ta promenna by mela byt ciste jen jmeno a zavorky presun k intu ... je to mozna zvuk z PB .. tu to bude fungovat, ale nedela se to

Reply:
Přehozeno, síla zvyku, ale to půjde :D